tests/framework/installer/ceph_manifests.go

```diff
@@ -45,6 +45,9 @@ type CephManifests interface {
 	GetObjectStoreUser(name, displayName, store, usercaps, maxsize string, maxbuckets, maxobjects int) string
 	GetBucketStorageClass(storeName, storageClassName, reclaimPolicy, region string) string
 	GetOBC(obcName, storageClassName, bucketName string, maxObject string, createBucket bool) string
+	GetOBCNotification(obcName, storageClassName, bucketName string, notificationName string, createBucket bool) string
+  GetBucketNotification(notificationName string, topicName string) string
+  GetBucketTopic(topicName string, storeName string, httpEndpointService string) string
```

Comment:
What's going on here with the spacing being different on the bottom two items?

Reply:
spaces instead of tabs - fixed